mobile_wallet/ios/build-ios.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+set -euxo pipefail
+
+(cd ../ && cargo build --release --target x86_64-apple-ios && \
```

Comment:
It is better to use `cargo --manifest-path ../Cargo.toml build --release --target ...` instead.